crates/cxx-qt-gen/src/generator/rust/property/mod.rs

```diff
@@ -161,6 +161,9 @@ mod tests {
             tokens_to_syn::<syn::Item>(quote! {
                 impl MyObjectQt {
                     pub fn set_trivial_property(mut self: Pin<&mut Self>, value: i32) {
+                        if self.rust().trivial_property == value {
```

Comment:
do we get a sane error if the type does not implement `PartialEq` ? eg if you try to run the `qml_features` and you didn't have `PartialEq` on one of the property types like `QString` what kind of error message do you get?

And i assume do pointer types already have PartialEq ? eg like `*mut T` ?

As i didn't know if somewhere we need to put a `T: PartialEq` or if the error message is sane enough anyway?

Reply:
(i also wasn't sure where this `T: PartialEq` would go, so if the current error message is sane that's fine :-)

Reply:
This is the error message you get when `PartialEq` is not implemented:

```
error[E0369]: binary operation `==` cannot be applied to type `cxx_qt_lib::QString`
  --> examples/cargo_without_cmake/src/cxxqt_object.rs:11:1
   |
11 | #[cxx_qt::bridge]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `cxx_qt::bridge` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0369`.
```

I think this error message is reasonable enough, right?

Reply:
Sane enough :-)